index.html

```diff
@@ -1379,10 +1382,10 @@ <h4>
         let action, text;
 
         for (const record of externalRecord.toRecords()) {
-          if (record.recordType == "text") {
+          if (record.recordType == "text" && !record.isLocal) {
             const decoder = new TextDecoder(record.encoding);
             text = decoder.decode(record.data);
-          } else if (record.recordType == "act") {
+          } else if (record.recordType == "act" && record.isLocal) {
```

Comment:
I think the isLocal test is not needed here. Only with those local types that clash with one of the web'ish type names we invented.

Reply:
"act" is an NFC Forum standardized local type.

Reply:
You're right!

Reply:
Fixed at https://github.com/w3c/web-nfc/pull/491/commits/53e66a2ece7c67bbbfeea9ae324a1cdbcbd9aff4